bank_statement_journal_items/README.rst

```diff
@@ -0,0 +1,4 @@
+Odoo bank statement journal items
+=================================
+
+This project allows to see Journal Items from bank statement.
```

Comment:
I thinks you should do a readme folder like you can see in https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

and after use the readme-generator https://github.com/OCA/maintainer-tools#readme-generator

Reply:
due to problems with dependencies i just created files under readme directory.